### PR TITLE
User score

### PR DESCRIPTION
**Pull request by SukritaEarn**
- Add status to tell where is the user current stage
- User can play and score will be saved to database
- Stage can lock left stages that user does not pass

Please review my code

### CODE REVIEW

**Comment by KasidisGit** [comment]
In similar html pattern, you can use base.html and extends for easier coding. :hankey:

**Comment by KasidisGit** [comment]
In similar html pattern, you can use base html and extends for easier coding. :hankey:

**Comment by SukritaEarn** [comment]
> In similar html pattern, you can use base.html and extends for easier coding. 💩

I fixed it, please review again. 🙇‍♀

File: game/templates/easy-question.html [hunk 1]
```diff
@@ -29,20 +29,17 @@
...
-{% if question.is_pass %}
+{% if user.current_easy > question.id %}
     <p><i class="fa fa-check"></i>  You got it!!</p>
+    {% if question.id < 15 %}
+        <button><a href="{% url 'game:e-picture' next_question.id %}">Next</a></button>
+    {% else %}
+        <button><a href="{% url 'game:easy' %}">Next</a></button>
+    {% endif %}
 {% else %}
     <p id="status"><i class="fa fa-close"></i>  Try again</p>
```
Comment (TirananE) [inline game/templates/easy-question.html:40]: Try again will always appear if user still not answer. You should try to make it appear only when user put their answer and submit. ❤️

File: game/templates/hard-question.html [hunk 2]
```diff
@@ -1,27 +1,48 @@
...
+{% if user.current_hard > question.id %}
+    <p><i class="fa fa-check"></i>  You got it!!</p>
+    {% if question.id < 15 %}
+        <button><a href="{% url 'game:h-picture' next_question.id %}">Next</a></button>
+    {% else %}
+        <button><a href="{% url 'game:hard' %}">Next</a></button>
+    {% endif %}
 {% else %}
-    <button><a href="{% url 'game:hard' %}">Next</a></button>
+    <p id="status"><i class="fa fa-close"></i>  Try again</p>
```
Comment (TirananE) [inline game/templates/hard-question.html:43]: This one also.

File: game/templates/medium-question.html [hunk 3]
```diff
@@ -1,24 +1,45 @@
...
+{% if user.current_medium > question.id %}
+    <p><i class="fa fa-check"></i>  You got it!!</p>
+    {% if question.id < 15 %}
+        <button><a href="{% url 'game:m-picture' next_question.id %}">Next</a></button>
+    {% else %}
+        <button><a href="{% url 'game:medium' %}">Next</a></button>
+    {% endif %}
 {% else %}
-    <button><a href="{% url 'game:medium' %}">Next</a></button>
+    <p id="status"><i class="fa fa-close"></i>  Try again</p>
```
Comment (TirananE) [inline game/templates/medium-question.html:40]: This one too.

**Comment by KasidisGit** [comment]
> > In similar html pattern, you can use base.html and extends for easier coding. 💩
>> I fixed it, please review again. 🙇‍♀

Thank for your work. Overall look's great should we merge? ❤️

File: game/templates/easy-question.html (hunk 1 above)
Comment (SukritaEarn) [reply game/templates/easy-question.html:40]: I will fix this after we merge with hint features.
Thank you for review 🙇‍♀

**Pull request merged**